provider/marathon/marathon_test.go

```diff
@@ -1801,3 +1807,33 @@ func TestParseIndex(t *testing.T) {
 		})
 	}
 }
+
+func TestMarathonGetBasicAuth(t *testing.T) {
+	provider := &Provider{}
+
+	applications := []struct {
```

Comment:
Given that we run two or three different test styles in `marathon_test.go`, things must already be pretty confusing. At this point, I'm only interested in reducing the amount of alignment effort new code brings along until I get my cleanup PR through.

We'll most likely move to `cases` everywhere in the not-too-remote future, so it's okay to adjust accordingly in your PR.